source/slang/slang-emit-c-like.cpp

```diff
@@ -1962,6 +1962,10 @@ void CLikeSourceEmitter::defaultEmitInstExpr(IRInst* inst, const EmitOpInfo& inO
         are hashed with 'getStringHash' */
         break;
 
+    case kIROp_undefined:
```

Comment:
Uninitialized local variables can get optimized out into an undefined value. We probably need to rethink whether it is a good idea to include such a hack here. 

Specifically, this is for the following case:
```
int GenericCompute<T:IInterface>(T obj, int inVal)
{
    GetterImpl getter;
	return obj.Compute(inVal) + T.Assoc.getBase(getter);
}
```
In this code, `getter` could be come `undefined`.